packages/next-common/hooks/fellowship/useFellowshipMemberParams.js

```diff
@@ -0,0 +1,16 @@
+import { useCallback, useState } from "react";
+import useSubStorage from "next-common/hooks/common/useSubStorage";
+
+export function useFellowshipMemberParams() {
```

Comment:
better rename to `useFellowshipParams` and with params `pallet = fellowshipCore`

aka `useFellowshipParams(pallet = 'fellowshipCore')`

Reply:
also rename this filename